common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -109,6 +105,160 @@ def test_courseware(self):
+            self.browser, self.course_id, entry_point='verify-now'
+        )
+        self.upgrade_page = PaymentAndVerificationFlow(self.browser, self.course_id, entry_point='upgrade')
+        self.fake_payment_page = FakePaymentPage(self.browser, self.course_id)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.problem_page = ProblemPage(self.browser)
+
+        # Add a verified mode to the course
+        ModeCreationPage(
+            self.browser, self.course_id, mode_slug=u'verified', mode_display_name=u'Verified Certificate',
+            min_price=10, suggested_prices='10,20'
+        ).visit()
+
+        # Auto-auth register for the course.
+        self._auto_auth(self.USERNAME, self.EMAIL, False)
+
+    def _auto_auth(self, username, email, staff):
+        """
+        Logout and login with given credentials.
+        """
+        AutoAuthPage(self.browser, username=username, email=email,
+                     course_id=self.course_id, staff=staff).visit()
+
+    def _login_as_a_verified_user(self):
+        """
+        login as a verififed user
+        """
+
+        self._auto_auth(self.USERNAME, self.EMAIL, False)
+
+        # the track selection page cannot be visited. see the other tests to see if any prereq is there.
+        # Navigate to the track selection page
+        self.track_selection_page.visit()
+
+        # Enter the payment and verification flow by choosing to enroll as verified
+        self.track_selection_page.enroll('verified')
+
+        # Proceed to the fake payment page
+        self.payment_and_verification_flow.proceed_to_payment()
+
+        # Submit payment
+        self.fake_payment_page.submit_payment()
+
+    def test_can_create_proctored_exam_in_studio(self):
+        """
+        Test that Proctored exam settings are visible in Studio.
+        """
+
+        # Given that I am a staff member
+        LogoutPage(self.browser).visit()
+        self._auto_auth("STAFF_TESTER", "staff101@example.com", True)
+
+        # When I visit the course outline page in studio.
+        self.course_outline.visit()
+
+        # And open the subsection edit dialog
+        self.course_outline.open_exam_settings_dialog()
+
+        # Then I can view all settings related to Proctored and timed exams
+        self.assertTrue(self.course_outline.proctoring_items_are_displayed())
+
+    def test_proctored_exam_flow(self):
+        """
+        Test that staff can create a proctored exam.
+        """
+
+        # Given that I am a staff member on the exam settings section
+        LogoutPage(self.browser).visit()
+        self._auto_auth("STAFF_TESTER", "staff101@example.com", True)
+        self.course_outline.visit()
+        self.course_outline.open_exam_settings_dialog()
+
+        # When I Make the exam proctored.
+        self.course_outline.make_exam_proctored()
+        time.sleep(2)  # Wait for 2 seconds to save the settings.
```

Comment:
Oh yea. Its a small popup at the bottom right of the screen. I had planned to fix it later. slipped.